front/lib/models/assistant/agent.ts

```diff
@@ -122,6 +123,10 @@ AgentConfiguration.init(
       type: DataTypes.INTEGER,
       allowNull: true,
     },
+    visualizationEnabled: {
```

Comment:
Should we store an array of string representing capabilities that don't require an action?

Reply:
Yeah fair 👍

Reply:
I am worried we're going to end up with an array for only one capability. I think we should wait until we need it before migrating

Reply:
Alright, it's up to you. It seems like it would save time to do it now, but I suppose you'll be the one to run this migration when the time comes 😛.